### ISSUE TITLE
Benchmark

### ISSUE THREAD

**Issue by Saiv46** [ProtoDef-io/node-protodefc#10]
Before replacing `node-protodef` in projects like NMP, we need to compare how faster it is than compiled JS

**Comment by rom1504** [comment]
Yeah that would have been needed. This project has been pretty much abandoned a few years back.
Are you interested in reviving it?

**Comment by Saiv46** [comment]
I'll try to revive it for other people and for my curiosity
